### PR TITLE
feat(tabs): Add TabPanel and accessibility

### PR DESCRIPTION
**Pull request by tor-martin-storsletten**
Bunch of changes to tabs, none of which should be breaking.

### CODE REVIEW

File: packages/tabs/src/component-tab.tsx [hunk 1]
```diff
@@ -49,7 +50,17 @@ export function Tab(props: TabProps) {
...
     return (
-        <button {...attrs} className={tab} onClick={handleClick}>
+        <button
+            type="button"
+            {...attrs}
+            role="tab"
+            aria-selected={isActive ? 'true' : 'false'}
+            aria-controls={isActive ? `fabric-tabpanel-${name}` : undefined}
+            id={`fabric-tab-${name}`}
+            tabIndex={isActive ? 0 : -1}
```
Comment (pearofducks) [inline packages/tabs/src/component-tab.tsx:60]: Is `tabIndex` here correct? I would think when a tab is active we don't care about if it's tab-able, but rather want to be able to navigate to the non-active tabs?

Comment (tor-martin-storsletten) [reply packages/tabs/src/component-tab.tsx:60]: Ah yes, good question. Switching between tabs using spatial navigation (arrow keys) is a standardized affordance. I.e. only the active element within a group is included in the tab order. This is true for other groups of elements too, such as radio buttons, grids, and dropdown menus.

Comment (pearofducks) [reply packages/tabs/src/component-tab.tsx:60]: Ah, I missed the key handling later on for this - my bad, thanks!

File: packages/tabs/src/component-tab-panel.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+    return (
+        <div
+            tabIndex={-1}
+            {...attrs}
+            role="tabpanel"
+            aria-labelledby={`fabric-tab-${name}`}
+            id={`fabric-tabpanel-${name}`}
+            hidden={
+                !document
```
Comment (benja) [inline packages/tabs/src/component-tab-panel.tsx:16]: This wouldn't render server-side as `document` isn't available on the server. We'll have to find another solution to check this.

Comment (benja) [reply packages/tabs/src/component-tab-panel.tsx:16]: One possible solution that comes to mind is setting a ref on the element, and checking the document inside a `useEffect`. If the document is in the dep array, it'll trigger when available afaik.

File: packages/tabs/src/component-tabs.tsx [hunk 3]
```diff
@@ -51,37 +56,108 @@ export function Tabs(props: TabsProps) {
...
-        setIsActive(name);
+    const change = (name) => {
+        setActive(name);
         updateWunderbar();
         onChange && onChange(name);
     };
 
+    const handleKeyDown = (event) => {
+        if (
+            !event.getModifierState() &&
```
Comment (pearofducks) [inline packages/tabs/src/component-tabs.tsx:99]: This isn't a valid use of `getModifierState` that I can see - it takes an argument to check if a specific modifier key is active.

https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/getModifierState

**Pull request merged**